patches/@embroider__router@2.1.7.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/dist/index.js b/dist/index.js
+index 2df582981f388c3cc8af7db9a2fb288710c4727c..2c4757d2dfd602ae2725a0cba01780e466d7565a 100644
+--- a/dist/index.js
```

Comment:
@void-mAlex why do we need this?

Reply:
we don't need it
it's was to prove work related to https://github.com/embroider-build/embroider/pull/1945
got that fixed upstream and will upgrade to that version and drop the patch